lib/containers/utils.pm

```diff
@@ -149,14 +149,29 @@ sub basic_container_tests {
 
 # Setup environment
 sub set_up {
-    my $dir = shift;
+    my ($dir, $file) = @_;
+    $file //= 'Dockerfile';
+    record_info "$file";
     die "You must define the directory!" unless $dir;
     assert_script_run("mkdir -p $dir/BuildTest");
     assert_script_run "curl -f -v " . data_url('containers/app.py') . " > $dir/BuildTest/app.py";
-    assert_script_run "curl -f -v " . data_url('containers/Dockerfile') . " > $dir/BuildTest/Dockerfile";
+    record_info('Dockerfile', script_output("cat $dir/BuildTest/app.py"));
+    assert_script_run "curl -f -v " . data_url("containers/$file") . " > $dir/BuildTest/Dockerfile";
+    record_info('Dockerfile', script_output("cat $dir/BuildTest/Dockerfile"));
     assert_script_run "curl -f -v " . data_url('containers/requirements.txt') . " > $dir/BuildTest/requirements.txt";
 }
 
+sub build_buildah_container_img {
+    my $dir = shift;
+    die "You must define the directory!" unless $dir;
+    my $runtime  = 'buildah';
+    my $registry = get_var('REGISTRY', 'docker.io');
+    assert_script_run("cd $dir/BuildTest");
+    record_info('Dockerfile', script_output('cat ./Dockerfile'));
```

Comment:
You do this on `set_up` already.

Reply:
there are but i would rather keep it simple and clean. i had enough with the if..else all around. In additional i would have to make extra tests. lets think the design principles later.

Reply:
right. i will remove it

Reply:
50e37222c